src/main/kotlin/no/nb/mlt/wls/order/controller/OrderController.kt

```diff
@@ -48,7 +49,7 @@ class OrderController(val orderService: OrderService) {
         )
     )
     @PostMapping("/order/batch/create")
-    fun createOrder(
+    suspend fun createOrder(
         @RequestBody payload: ApiOrderPayload
     ): ResponseEntity<ApiOrderPayload> = orderService.createOrder(payload)
```

Comment:
Correct me if I'm wrong should the return type be changed here to something more reactive, or do we need to return ResponseEntity

Reply:
This depends on what we want regarding potential/future consumers of our own API. If we want to be able to build the service to act in a reactive manner, meaning sending partial data or streaming the data, then the answer is yes. Otherwise a `ResponseEntity` should be fine for now.